cmd/firenear/cli/common-flag.go

```diff
@@ -22,26 +22,27 @@ import (
 
 func init() {
 	launcher.RegisterCommonFlags = func(_ *zap.Logger, cmd *cobra.Command) error {
+
 		//Common stores configuration flags
-		cmd.Flags().String("common-blocks-store-url", MergedBlocksStoreURL, "[COMMON] Store URL (with prefix) where to read/write. Used by: relayer, fluxdb, trxdb-loader, blockmeta, search-indexer, search-live, search-forkresolver")
-		cmd.Flags().String("common-oneblock-store-url", OneBlockStoreURL, "[COMMON] Store URL (with prefix) to read/write one-block files. Used by: mindreader, merger")
-		cmd.Flags().String("common-forkedblocks-store-url", OneBlockStoreURL, "[COMMON] Store URL (with prefix) to read/write one-block files. Used by: mindreader, merger")
-		cmd.Flags().String("common-blockstream-addr", RelayerServingAddr, "[COMMON] gRPC endpoint to get real-time blocks. Used by: fluxdb, trxdb-loader, blockmeta, search-indexer, search-live (relayer uses its own --relayer-blockstream-addr)")
+		cmd.Flags().String("common-merged-blocks-store-url", MergedBlocksStoreURL, "[COMMON] Store URL (with prefix) where to read/write merged blocks.")
+		cmd.Flags().String("common-one-block-store-url", OneBlockStoreURL, "[COMMON] Store URL (with prefix) to read/write one-block files.")
+		cmd.Flags().String("common-forked-blocks-store-url", OneBlockStoreURL, "[COMMON] Store URL (with prefix) to read/write index files.")
+		cmd.Flags().String("common-live-blocks-addr", RelayerServingAddr, "[COMMON] gRPC endpoint to get real-time blocks.")
 
 		cmd.Flags().Bool("common-atm-cache-enabled", false, "[COMMON] enable ATM caching")
```

Comment:
I just realized also that all the flags starting with `common-atm-...` should be updated also, use https://github.com/streamingfast/firehose-acme/blob/master/cmd/fireacme/cli/common-flag.go#L30 (the one starting with `common-blocks-cache`)